rosdep/base.yaml

```diff
@@ -1041,6 +1041,17 @@ fakeroot:
   nixos: [fakeroot]
   rhel: [fakeroot]
   ubuntu: [fakeroot]
+fast-float:
+  arch: [fast_float]
+  debian: [libfast-float-dev]
+  fedora: [fast_float-devel]
+  gentoo: [dev-cpp/fast_float]
+  nixos: [fast-float]
+  osx:
+    homebrew:
+      packages: [fast_float]
+  opensuse: [fast_float]
```

Comment:
The openSUSE package listed couldn't be found by the automation, as it appears from the dashboard that the package is only available in openSUSE Tumbleweed. The rule for openSUSE should be omitted until it is available in a supported release.

```suggestion
```